app/src/main/java/com/github/sdp/tarjetakuna/database/CardsRTDB.kt

```diff
@@ -0,0 +1,102 @@
+package com.github.sdp.tarjetakuna.database
+
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseReference
+import com.google.gson.Gson
+import java.util.concurrent.CompletableFuture
+
+/**
+ * This class is used to manage the global collection of cards contained by all users (no duplicates).
+ */
+class CardsRTDB(database: Database) {
+
+    private var db: DatabaseReference
+
+    init {
+        this.db = database.cardsTable() //Firebase.database.reference.child("cards")
+    }
+
+    /**
+     * Add a card to the global card collection.
+     */
+    fun addCardToGlobalCollection(fbCard: DBMagicCard) {
+        val cardUID = fbCard.code + fbCard.number

```

Comment:
for this uid, can you do code + "_" + number ? add the dash in the middle ? it could (unlikely) happen that we have MT15 + 45 or MT1 + 545, that would give the same uid for 2 diff card